.github/workflows/main.yml

```diff
@@ -0,0 +1,26 @@
+on: [push, pull_request]
+name: Test
+jobs:
+  test:
+    strategy:
+      fail-fast: false
+      matrix:
+        os: [ubuntu-latest, windows-latest]
+    runs-on: ${{matrix.os}}
+    steps:
+    - uses: actions/setup-go@v2
+      with:
+        go-version: 1.x
+    - run: |
+        go get golang.org/x/tools/cmd/goimports
+        go install github.com/mattn/goveralls@latest
+    - run: |
+        git config --global core.symlinks true
+    - uses: actions/checkout@v2
+    - if: ${{ runner.os == 'Linux' }}
+      run: if [ $(goimports -l .) ]; then goimports -d .; echo 'Failed the goimports format check. Please format the code using "goimports -w ."'; exit 1; fi
+    - run: go test -mod=mod -coverprofile=covprofile ./...
+    - env:
+        COVERALLS_TOKEN: ${{ secrets.github_token }}
```

Comment:
Just curious: Where is this token set up? It would be great if we could document it somewhere (Either here or in the internal team site works)

Reply:
It's provided by GitHub itself: https://docs.github.com/en/actions/reference/authentication-in-a-workflow#about-the-github_token-secret